src/main/java/net/atos/zac/app/zaken/model/RESTZaak.java

```diff
@@ -11,22 +11,27 @@
 import java.util.Map;
 import java.util.UUID;
 
+import jakarta.validation.constraints.NotNull;
+
 import net.atos.zac.app.identity.model.RESTGroup;
 import net.atos.zac.app.identity.model.RESTUser;
 import net.atos.zac.app.klanten.model.klant.IdentificatieType;
 import net.atos.zac.app.policy.model.RESTZaakRechten;
 import net.atos.zac.zoeken.model.ZaakIndicatie;
 
+
 public class RESTZaak {
 
     public UUID uuid;
 
     public String identificatie;
 
-    public String omschrijving;
+    @NotNull(message = "Omschrijving is required")
+    public String omschrijving; // Assuming this is the description field
```

Comment:
The comment should be removed I think?
```suggestion
    public String omschrijving;
```